src/components/NumberGenerator.js

```diff
@@ -0,0 +1,12 @@
+export class NumberGenerator {
+  constructor(minimum, rangeOfGeneratedNumbersValue) {
+    this.min = minimum;
+    this.max = rangeOfGeneratedNumbersValue;
```

Comment:
może wystarczy min i max? Nazwy powinny być jak najprostsze i jak najkrótsze żeby się to łatwo czytało